featuretools/tests/entityset_tests/test_serialization.py

```diff
@@ -16,14 +16,21 @@
     find_variable_types
 )
 
+SCHEMA_VER = serialize.SCHEMA_VERSION
 CACHE = os.path.join(os.path.dirname(integration_data.__file__), '.cache')
 BUCKET_NAME = "test-bucket"
 WRITE_KEY_NAME = "test-key"
 TEST_S3_URL = "s3://{}/{}".format(BUCKET_NAME, WRITE_KEY_NAME)
-S3_URL = "s3://featuretools-static/test_serialization_data_1.0.0.tar"
-URL = 'https://featuretools-static.s3.amazonaws.com/test_serialization_data_1.0.0.tar'
+
+S3_URL = "s3://featuretools-static/test_serialization_data_{}.tar".format(SCHEMA_VER)
```

Comment:
this file shouldn't be based off of the schema version since it might change with the schema change. if anything it should be based on the featuretools version. but really, i would just say, let's manually increment it as needed